pkg/asset/targets/targets.go

```diff
@@ -66,6 +66,9 @@ var (
 		&cluster.TerraformVariables{},
 		&kubeconfig.AdminClient{},
 		&password.KubeadminPassword{},
+		&machine.Master{},
+		&machine.Worker{},
+		&bootstrap.Bootstrap{},
```

Comment:
What are these additions for?

Reply:
Because when we use command `bin/openshift-install create cluster`, the master.ign, bootstrap.ign and worker.ign will not be generated in form of file,  these additions in targets.go tell installer to generate them, so that iso file can be created.

Reply:
I used this and only see following , but master ign should be very big?  XXXX below is the sshkey I am having..

{"ignition":{"config":{"append":[{"source":"https://api-int.test1.aa.testing:22623/config/master","verification":{}}]},"security":{"tls":{"certificateAuthorities":[{"source":"data:text/plain;charset=utf-8;base64,XXXXXXXXXXXXXXXXXXXXX","verification":{}}]}},"timeouts":{},"version":"2.2.0"},"networkd":{},"passwd":{},"storage":{},"systemd":{}}

Reply:
@jichenjc 
I think what you said may be bootstrap.ign instead of master.ign. 
I run installer in the original way, below is master ignition's details in terraform.tfvars.json.
terraform.tfvars.json contains all info installer tells terraform, and master ignition is small as yours.

```
  "ignition_master": "{\"ignition\":{\"config\":{\"append\":[{\"source\":\"https://api-int.test1.tt.testing:22623/config/master\",\"verification\":{}}]},\"security\":{\"tls\":{\"certificateAuthorities\":[{\"source\":\"data:text/plain;charset=utf-8;base64,XXXXXXXXXX",\"verification\":{}}]}},\"timeouts\":{},\"version\":\"2.2.0\"},\"networkd\":{},\"passwd\":{},\"storage\":{},\"systemd\":{}}"
```